WalletWasabi.Fluent/ApplicationStateManager.cs

```diff
@@ -137,7 +137,8 @@ private void CreateAndShowMainWindow()
 			{
 				// _hideRequest flag is used to distinguish what is the user's intent.
 				// It is only true when the request comes from the Tray.
-				if (Services.UiConfig.HideOnClose || _hideRequest)
+				if ((Services.UiConfig.HideOnClose && App.EnableFeatureHideOnClose)
+				    || _hideRequest)
```

Comment:
Shouldn't it be like this:
```suggestion
				if ((Services.UiConfig.HideOnClose || _hideRequest) && App.EnableFeatureHideOnClose)
```

It looks safer, as with the previous solution if _hideRequest is true, it will just hide, but we should prevent hiding completely.